src/reprojection.cpp

```diff
@@ -1,74 +1,59 @@
-/* reprojection.c
- *
- * Convert OSM coordinates to another coordinate system for
- * the database (usually convert lat/lon to Spherical Mercator
- * so Mapnik doesn't have to).
- */
-
-#include <cstdio>
-#include <cstdlib>
-#include <cstring>
-
 #include <osmium/geom/mercator_projection.hpp>
 
 #include "reprojection.hpp"
 
-/** must match expire.tiles.c */
-#define EARTH_CIRCUMFERENCE 40075016.68
-
 namespace {
 
-void latlon2merc(double *lat, double *lon)
+Coordinates lonlat2merc(Coordinates coords)
```

Comment:
Shouldn't it rather be coord2merc now?

Reply:
It still converts coordinates from lon/lat format to Mercator. The `Coordinates` just holds two doubles and can be used by many different coordinate systems.